src/ProjectTemplates/Shared/TemplatePackageInstaller.cs

```diff
@@ -110,7 +110,7 @@ private static async Task InstallTemplatePackages(ITestOutputHelper output)
         foreach (var packagePath in builtPackages)
         {
             output.WriteLine($"Installing templates package {packagePath}...");
-            var result = await RunDotNetNew(output, $"--install \"{packagePath}\"");
+            var result = await RunDotNetNew(output, $"install \"{packagePath}\"");
```

Comment:
Huh❔

Reply:
`--install` is deprecated and emits a warning now. This is what it said to use instead.

Reply:
<img width="623" alt="image" src="https://user-images.githubusercontent.com/249088/173682013-006ff43d-822d-4fff-9d07-480ce78bd5c7.png">